src/client/components/app/LineDetailsDisplay/LineMetadata.tsx

```diff
@@ -73,6 +78,40 @@ const DataMetric = styled.h5`
   `}
 `;
 
+const MetadataBox = styled.div`
+  ${({ theme }) => `
+    font-size: ${theme.fonts.sizes.md};
+  `}
+`;
+
+const MetadataTitle = styled.span`
+  ${({ theme }) => `color: ${theme.colors.primary}; `}
+`;
+
+const StatusWithColor = styled.span<{ status: string }>`
+  color: ${({ status }) => {
+    console.log('stats color', status);
+    switch (status) {
+      case UNINITIALIZED_STATUS:
+        return '#E6E600'; // darkish yellow
+      case ACTIVE_STATUS:
+      case REPAID_STATUS:
+        return '#6AFF4D'; // light green
+      case INSOLVENT_STATUS:
+      case LIQUIDATABLE_STATUS:
+        return '#FF1919'; // bright red
+    }
+  }};
+`;
+
+const CratioWithColor = styled.span<{ diff: number }>`
+  color: ${({ diff }) => {
+    if (diff >= 15) return '#6AFF4D'; // decent margin - light green
```

Comment:
Do we want to match these to colors in `theme` instead of hardcoding the colors? For if user switches modes (dark, light, etc.)?